### ISSUE TITLE
Please update lager

### ISSUE THREAD

**Issue by deadtrickster** [inaka/elvis#385]
Looks like lager version is a bit outdated and elvis fails to compile on otp19.

see https://github.com/basho/lager/pull/321

**Comment by elbrujohalcon** [comment]
@deadtrickster would you mind sending a PR with the change?

**Comment by deadtrickster** [comment]
@elbrujohalcon I noticed lager in elvis_core outdated too (and lager is used by elvis_core not by elvis itself). I made branch with fix - https://github.com/deadtrickster/elvis_core/tree/update-deps.
However looks like lager still should be explicitly listed as dependency in elvis because lots of other libs use outdated versions too. 
Question: should I leave elvis_core as is and just update lager in elvis, or maybe submit pr to elvis_core first and then submit pr here with update lager and elvis_core ref?
